src/middlewared/middlewared/plugins/system.py

```diff
@@ -485,7 +486,12 @@ def _get_license():
             "customer_name": licenseobj.customer_name,
             "expired": licenseobj.expired,
             "features": [],
-            "addhw": licenseobj.addhw,
+            "addhw_raw": licenseobj.addhw,
```

Comment:
Thats backward incompatible.
The new one should probably be "addhw_detail" or something?